### DISCUSSION TITLE
Integration Defectgojo with Codacy

### DISCUSSION THREAD

**Discussion by guedesanspace** [DefectDojo/django-DefectDojo#8970] [General]
Hello,

I'd like to know if it is possible to do integration between codacy and defectdojo?

Any ideia?

**Comment by mtesauro** [comment]
@guedesanspace As long as a security tool produces something like DefectDojo Findings aka security issues, then it's possible to get that data into DefectDojo.

The easiest way to do an integration with any security tool is to write a parser for it - see https://documentation.defectdojo.com/contributing/how-to-write-a-parser/

If you're not a Python coder, you can greatly increase the chance that someone will write one for you by adding an example output (sanitized) to this issue. We'll need a 'normal' output example with several different kinds of findings - the better the variety the better they parser will be. For writing unit-tests, we'll also need the following output examples (these can be trimmed down from the original):
* many findings
* no findings
* one finding

**Reply by manuel-sommer** [reply]
I can help you with a PR. Can you please add the finding files here: https://github.com/DefectDojo/django-DefectDojo/issues/9068 ?

**Reply by manuel-sommer** [reply]
Reminder @guedesanspace

**Reply by manuel-sommer** [reply]
Reminder @guedesanspace, could you provide codacy sample files, or otherwise, I will close the issue again.

**Reply by manuel-sommer** [reply]
@mtesauro could you close this discussion as there was no response from @guedesanspace ?

I will also close the ticket.
